setup.py

```diff
@@ -1,18 +1,10 @@
-from __future__ import (absolute_import, division, print_function)
-__metaclass__ = type
-
-import pathlib
+from __future__ import absolute_import, division, print_function
 
 from setuptools import find_packages, setup
 
-here = pathlib.Path(__file__).parent.resolve()
-
-install_requires = (here / 'requirements.txt').read_text(encoding='utf-8').splitlines()
-
 setup(
-    install_requires=install_requires,
-    package_dir={'': 'lib',
-                 'ansible_test': 'test/lib/ansible_test'},
+    install_requires=open('requirements.txt', encoding='utf-8').read().splitlines(),
+    package_dir={'': 'lib', 'ansible_test': 'test/lib/ansible_test'},
```

Comment:
Does this change improve anything? How a long line is better than clean separate ones? Please, don't include this in the patch.